index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <header id="header">
+      <div class="header-container">
+        <div class="logo">
+          <img class="logo-image" id="header-img" src="./images.PNG">
+        </div>
+        <div class="menu-items">
+          <nav id="nav-bar">
+            <ul>
+              <li><a class="nav-link" href="#">Home</a></li>
+              <li><a class="nav-link" href="#">About</a></li>
+              <li><a class="nav-link" href="#">Contact</a></li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+      
+    </header>
+
```

Comment:
It would be great if you remove the unwanted blank lines and indentation to make your code look more professional.